app/views/partials/_sidebar.html.erb

```diff
@@ -3,8 +3,8 @@
     <div class="sidebar-content">
       <ul>
         <li><a href="#"><h4>Relevant Jobs</h4></a></li>
-        <li><a href="#"><h4>My Jobs</h4></a></li>
-        <li><a href="#"><h4>All Jobs</h4></a></li>
+        <li><a href= <%= jobs_path(my_jobs: 'my_jobs') %>><h4>My Jobs</h4></a></li>
```

Comment:
I was just wondering if we were going to be using AngularJS in this situation maybe if it would be easier to leave it as href?